src/debugAdapter/goDebug.ts

```diff
@@ -245,6 +245,28 @@ function normalizePath(filePath: string) {
 	return filePath;
 }
 
+function fileContainsEntryPoint(filePath: string) {
+	let goOutline = getBinPathWithPreferredGopath('go-outline', []);
+	let goOutlineFlags = ['-f', filePath];
+
+	try {
+		let goOutlineOutput = execFileSync(goOutline, goOutlineFlags);
+		let declarations = JSON.parse(goOutlineOutput) as { label, type, children }[];
```

Comment:
I opted for an anonymous type here since we don't need the full `GoOutlineDeclaration` information as defined in `goOutline`. An alternative if we wanted to keep the type information in sync for future updates could be moving `GoOutlineDeclaration` and `GoOutlineRange` into a separate file to share with `goDebug`.